src/store/ui/ui.reducers.ts

```diff
@@ -16,8 +16,8 @@ export interface UIState {
 
 const initialState: UIState = {
   theme: 'dark',
-  showChatbar: true,
-  showPromptbar: true,
+  showChatbar: false,
+  showPromptbar: false,
```

Comment:
Do we really want to hide these panel by default?

Reply:
I think it's better to hide, because without them opened it looks ok on mobile and desktop, but when they both opened by default it looks very bad on mobile and ok on desktop  :)
![image](https://github.com/epam/ai-dial-chat/assets/6322751/1ddd7447-6d0e-4463-9271-241596c82d35)